src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
I see, ok.  So, just to make sure I'm understanding correctly, essentially if the protocols value comes in as SslProtocols.None, we should just let libcurl do whatever it wants to do; otherwise, we continue doing what we're doing today, correct?